### DISCUSSION TITLE
What font or other config to render the `Snippet` icon ""

### DISCUSSION THREAD

**Discussion by SichangHe** [LazyVim/LazyVim#2863] [Q&A] [answered]
![image](https://github.com/LazyVim/LazyVim/assets/84777573/1bfb11c2-ab27-4a04-8ed8-e5e5a47df1b1)

Please notice, at the bottom, the icon for `Snippet` is not rendered, probably because my font "JetBrains Mono Nerd Font" does not have it, although it is already a nerd font.

**Comment by SichangHe** [comment]
Icon source: https://github.com/LazyVim/LazyVim/blob/cfbd3582736286433ee5532e1ea3a8d05a1e2649/lua/lazyvim/config/init.lua#L84

**Reply by folke** [reply]
It's in the docs
https://www.lazyvim.org/#%EF%B8%8F-requirements

**Reply by folke** [reply, marked as answer]
And that icon in your screenshot is actually the correct icon :)

**Reply by SichangHe** [reply]
My bad. What a brainfart… Just a square icon. 🙃
